src/Selenium2Driver.php

```diff
@@ -434,75 +370,71 @@ public function switchToIFrame($name = null)
     public function setCookie($name, $value = null)
     {
         if (null === $value) {
-            $this->wdSession->deleteCookie($name);
+            $this->webDriver->manage()->deleteCookieNamed($name);
 
             return;
         }
 
-        $cookieArray = array(
-            'name'   => $name,
-            'value'  => urlencode($value),
-            'secure' => false, // thanks, chibimagic!
-        );
-
-        $this->wdSession->setCookie($cookieArray);
+        $cookie = new Cookie($name, \urlencode($value));
+        $this->webDriver->manage()->addCookie($cookie);
     }
 
     /**
      * {@inheritdoc}
      */
     public function getCookie($name)
     {
-        $cookies = $this->wdSession->getAllCookies();
-        foreach ($cookies as $cookie) {
-            if ($cookie['name'] === $name) {
-                return urldecode($cookie['value']);
-            }
+        $cookie = $this->webDriver->manage()->getCookieNamed($name);
+        if (!$cookie) {
+            return null;
         }
+
+        return \urldecode($cookie->getValue());
     }
 
     /**
      * {@inheritdoc}
      */
     public function getContent()
     {
-        return $this->wdSession->source();
+        $source = $this->webDriver->getPageSource();
+        return str_replace(array("\r", "\r\n", "\n"), \PHP_EOL, $source);
```

Comment:
I would rather return them with `"\n"` rather than `\PHP_EOL` if we normalize it

Reply:
IIRC if you try to execute phpunit on Windows, it will fail on multi-line assert as example: `\Behat\Mink\Tests\Driver\Form\Html5Test::testHtml5FormButtonAttribute`

Reply:
Well, I consider it easier to write tests when I don't need to change the newline chars in the test depending on the OS.

One reason it might fail for you is if your git is configured to convert the code to CRLF on checkout. But I don't consider that a good setup. The unmodified source code must be working.